ocpp1.6_test/cancel_reservation_test.go

```diff
@@ -42,7 +42,7 @@ func (suite *OcppV16TestSuite) TestCancelReservationE2EMocked() {
 	cancelReservationConfirmation := reservation.NewCancelReservationConfirmation(status)
 	channel := NewMockWebSocket(wsId)
 
-	reservationListener := MockChargePointReservationListener{}
+	reservationListener := &MockChargePointReservationListener{}
```

Comment:
I changed I think most of these to have the same pattern of "declare struct as a pointer" to avoid the copied lock problem.